### PR TITLE
MDN CSS: Add coverage data

### LINKED ISSUE #510: MDN CSS: add coverage data
### Description
For every Fathead, we need to gather the set of articles we wish to provide coverage for.
Create a `cover/` directory containing text files that list all the topic titles and language features to cover. 
These lists will then be used as Unit tests to help you measure the Fathead's coverage. 
For more info please see [the docs](https://docs.duckduckhack.com/programming-mission/creating-effective-fatheads.html).

## People to notify
@hchienjo 

## Get Started
- [x] 1) Claim this issue by commenting below
- [ ] 2) Review our [Contributing Guide](https://github.com/duckduckgo/zeroclickinfo-fathead/blob/master/CONTRIBUTING.md)
- [ ] 3) [Set up your development environment](https://docs.duckduckhack.com/welcome/setup-dev-environment.html), and fork this repository
- [ ] 4) Create a Pull Request

## Resources
- Join [DuckDuckHack Slack](https://quackslack.herokuapp.com/) to ask questions
- Join the [DuckDuckHack Forum](https://forum.duckduckhack.com/) to discuss project planning and Instant Answer metrics
- Read the [DuckDuckHack Documentation](https://docs.duckduckhack.com/) for technical help

---

Instant Answer Page: https://duck.co/ia/view/mdn_css

### PR DESCRIPTION
**Pull request by hchienjo**
## Description of new Instant Answer, or changes

Adds text file containing CSS keywords that this IA aims to provide coverage for.

## Related Issues and Discussions

Fixes #510

## People to notify

@moollaza 

---

Instant Answer Page: https://duck.co/ia/view/mdn_css

This is still WIP.

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

**Comment by hchienjo** [comment]
This is now complete. Needs review. @duckduckgo/community-leaders

**Comment by gautamkrishnar** [comment]
👍 Looking great...

File: lib/fathead/mdn_css/cover/keywords.txt [hunk 1]
```diff
@@ -0,0 +1,1203 @@
...
+drop-initial-before-adjust
+drop-initial-before-align
+drop-initial-size
+drop-initial-value
+element()
+em
+:empty
+empty-cells
+:enabled
+-epub-
```
Comment (moollaza) [inline lib/fathead/mdn_css/cover/keywords.txt]: @hchienjo where does this come from? I don't believe we expect to cover this, so we can remove this and the others that are similar (e.g. `-atsc-`)

**Comment by moollaza** [comment]
@hchienjo using the `zaahir/test-all-fatheads` branch I was able to run the tests and it shows that our output.txt doesn't cover about 700 of these.

Can you look through the 700 that aren't covered and remove the ones we won't be able to? i.e. the stuff that MDN doesn't support? For now we may need to remove the -webkit and -moz until we can expect to cover them all

It also looks like several of these are expected to be prefixed with `-webkit` but aren't in the list (e.g. `border-before-color` -- that looks like it's webkit only?)

File: lib/fathead/mdn_css/cover/keywords.txt (hunk 1 above)
Comment (hchienjo) [inline lib/fathead/mdn_css/cover/keywords.txt]: Alright. Looking into it.

**Comment by hchienjo** [comment]
It's -epub- as an entry on its own that is problematic. I am taking care 
of that. The complete epub in the output.txt should not be a problem.

**Comment by moollaza** [comment]
@hchienjo I'm saving your original list in a Gist in case we need to pul stuff out of that later: 

https://gist.github.com/moollaza/58c8ad79d91d472c1c2e112aaa510165

This is a great start. I think we still have a bit of work, but the remaining failing articles all seem to be legitimate cases of articles we should have and don't, because they exist on MDN 👍

**Comment by hchienjo** [comment]
@moollaza sorry I wasn't live to respond to your changes in a sooner. It's usually dead of night when I am live and vice versa when you are. Really appreciate that you took the liberty to make the changes.

**Pull request merged**